src/wymeditor/editor/gecko.js

```diff
@@ -146,7 +146,8 @@ WYMeditor.WymClassGecko.prototype._designModeQuirks = function () {
     try {
         wym._doc.execCommand("styleWithCSS", '', false);
         wym._doc.execCommand("enableObjectResizing", false, false);
-        wym._doc.execCommand("enableInlineTableEditing", false, false);
```

Comment:
This enables the by-default Firefox's native table editing thingo, which looks like this:
![screenshot_2015-11-21_20-06-32](https://cloud.githubusercontent.com/assets/635591/11319778/8769be2a-908b-11e5-99c7-9dfb37d6d967.png)
It works quite alright, it seems, during some tinkering.